docs/deployment/k4k8s-enterprise.md

```diff
@@ -60,11 +60,11 @@ to log in to Bintray and password
 is an API-key that can be provisioned via Bintray.
 
 ```bash
-$ kubectl create secret -n kong docker-registry kong-enterprise-k8s-docker \
-    --docker-server=kong-docker-kong-enterprise-k8s.bintray.io \
+$ kubectl create secret docker-registry kong-enterprise-edition-docker \
```

Comment:
Why remove `-n kong` here? I understand that this doc instructs the user to deploy Kong in the `kong` namespace. If the user blindly copy-pastes this command, it will create the secret in the default namespace (or one set in the user's kubeconfig context) and lead to confusion and an `ImagePullBackOff`.

Follow-up question if we still want to remove `-n kong`: does the comment _Again, please take a note of the namespace `kong`._ still make sense after this change?

Reply:
Inadvertent copy-pasta clobbered it, I think. Not all of our docs use that namespace convention. Have restored it, good catch.